config/ide-helper.php

```diff
@@ -89,6 +89,9 @@
     | Define in which directories the ide-helper:models command should look
     | for models.
     |
+    | glob patterns are supported to easier reach models in sub-directories,
+    | e.g. `app/Services/* /Models` (without the space)
```

Comment:
So why the space?

Reply:
Yeah well :) because otherwise it would end the phpdoc 💥

Reply:
And I tried (maybe too hard) to give a useful example

Reply:
hehe good point.